narwhals/_arrow/namespace.py

```diff
@@ -325,6 +325,11 @@ def mean(self, *column_names: str) -> ArrowExpr:
             *column_names, backend_version=self._backend_version, dtypes=self._dtypes
         ).mean()
 
+    def skew(self, *column_names: str) -> ArrowExpr:
+        return ArrowExpr.from_column_names(
+            *column_names, backend_version=self._backend_version, dtypes=self._dtypes
+        ).skew()
+
```

Comment:
Polars does not have a `skew` function in the high level namespace, so I don't think we should either

Reply:
Good one! Was not aware that this was a separate thing. Removed the `skew` functions in the `namespace.py` files.